projects/packages/jetpack-mu-wpcom/changelog/add-comment-likes-to-wpcom

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: added
+
+Added a like quick link on each comment when using wp-admin interface.
```

Comment:
Should this be edited to note that this is only for Simple sites for now? (If so, we need to edit the PR's title and description too).